pyproject.toml

```diff
@@ -30,7 +30,9 @@ dependencies = [
     "biocommons.seqrepo",
     "gene-normalizer ~=0.10.0",
     "civicpy ~=5.0",
-    "cool-seq-tool ~=0.15.1"
+    "cool-seq-tool ~=0.15.1",
+    "ipykernel",
+    "jupyter",
```

Comment:
These shouldn't be required dependencies. Consider making an optional notebooks group